### ISSUE TITLE
[Epic] Content and IA Request for Vitals domain from mhv-medical-records

### ISSUE THREAD

**Issue by alexiawunder** [department-of-veterans-affairs/va.gov-team#55308]
## What does your team need support for? Check all that apply.
- [X] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Something else 
 
Note: Based on conversations with Laura and Kristin, this is a content-only ticket. Kristin will combine all of the domains into one IA ticket, but all of the domains will be broken out into individual content tickets. All of medical records will go through the regular collab cycle ticket together.

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [X] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
October 2024
  
*Please provide an estimated date so our team can create other relevant tickets.*
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: [Medical Records Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records)
- Link to or attach [user flow](https://www.sketch.com/s/0803544d-e78c-423a-a70e-7db01a9a31c7/a/uuid/FBE40EA8-83CC-41A6-ABDE-F16F351B0FCF)
- Link to [Domain Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/data-domains/vitals/vitals-brief.md)
- Link to prototype or mockups:
  - Vitals list view [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/4E480FBD-C72A-435B-8F8B-CB8031A673AA) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/5ACFEE0F-2FFC-42AB-BFD9-C1E69FBACE4E)
  - Details page variations:
    - Vitals/Blood pressure [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/BB8932F5-531E-4CB2-B66E-24B194BAD904) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/6210A759-096D-4821-BA3B-EB79DCC5164A)
    - Vitals/Height [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/367F815C-AEAD-4911-8C4B-AC5BED825C58) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/1F057337-F44E-4D98-9DB0-43477F5E5ADD)
    - Vitals/Pain level [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/19939DB4-47DF-4C38-95A6-31157D604B4F) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/159A2770-70EA-4324-BC78-D716A22B581D)
    - Vitals/Pulse Rate [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/7633BF05-6505-480C-B5AE-57FD34F70FFE) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/85424C3F-1BBC-4D69-8B8E-89335D2200A9)
    - Vitals/Respiration [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/DA2D624B-C3F1-437C-9341-6F76E2698D4C) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/9E12E49B-E6EB-4451-8E2D-C3FC3340D8D1)
    - Vitals/Temperature [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/34076153-8072-4D31-81E4-17EC4207D2FD) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/67E54149-D1FA-4BFB-985E-976B889B323B)
    - Vitals/Weight [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/1558EE8F-3C1D-4974-9061-586F45B7FFA2) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/0C6A591B-364A-4662-A56C-D99BC2298D3F)
    - Vitals/PDF (download view) [Page 1](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/4AD1388E-9E8E-4C5A-9902-9342D9CC8F2C) | [Page 2](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/800FB665-6802-4FAA-891A-BCFAADEF008B)

- Link to any epics/issues that may be helpful: N/A

## Will this work be going through the Collaboration Cycle?
- [X] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: ~June 2023
- Estimated staging review date:~June 2023
- Content and IA work needed by: We'd like to conduct usability testing with approved content at the end of March, 2023

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [X] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: mhv-medical-records
- OCTO-DE product owner: Lauren Alexanderson
- Product manager: Bryan Riley
- Designer: Alexia Wunder, Bobby Bailey
- FE engineer: Mike Moyer
- Product/team Slack channel: #mhv-medical-records

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by laurwill** [comment]
@alexiawunder @BobbyBaileyRB ,
Updates for vitals:
- Intro for list view and "what to know" in [Medical records working content draft](https://dvagov-my.sharepoint.com/:w:/g/personal/laura_willwerth_va_gov/Eacz1GNGqrdNn3b5gC1j0n4BjNyZ6LyafJCpEfebp-cPVg?e=f6e3Hy)
- Comments on Sketch detail screens (mobile versions) have suggested PL units for each vital and suggested intro text for the detail screens. 
- Data field labels in github tables, including PL suggestions for the names of each vital type

**Remaining questions:**
- Is pain level always on a scale of 1 to 10? And is there anything else we need to say about what pain level indicates?

**Comment by bryan-riley-va** [comment]
No longer needed
